iOS/MasterTimeline/Cell/MasterTimelineDefaultCellLayout.swift

```diff
@@ -11,10 +11,10 @@ import RSCore
 
 struct MasterTimelineDefaultCellLayout: MasterTimelineCellLayout {
 
-	static let cellPadding = UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16)
+	static let cellPadding = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 16)
 	
-	static let unreadCircleMarginLeft = CGFloat(integerLiteral: 8)
-	static let unreadCircleDimension = CGFloat(integerLiteral: 8)
+	static let unreadCircleMarginLeft = CGFloat(integerLiteral: 0)
```

Comment:
Technically this isn't needed anymore. I can get rid of it if people think it's just dead code. I left it around in case we want to tweak the value in the future.